releasenotes/notes/deprecate-monkey-functions-01f8080ab711ae74.yaml

```diff
@@ -0,0 +1,12 @@
+---
+prelude: >
+     The monkey.patch_module as well as the monkey.get_patch_module has been deprecated.
+
```

Comment:
```suggestion
```

We don't need this, that one example I shared you had it because it was a bunch of changes all together, but we don't need it here.